msr-flows/readme.md

```diff
@@ -1,50 +1,11 @@
 These scripts are used to analyze flows in MSRs.
 
-### Short Term Testing
+## Flow Tests
 
-Testing to see the short term effects cycle time has upon msr behaviour,
```

Comment:
I assume these deletions are to reflect the refactor?

Reply:
Yeah, I was using the original scripts for testing different things, so the refactor also trimmed some of those.

Reply:
Okay. Marking as resolved